package.json

```diff
@@ -16,40 +20,39 @@
     "prepublishOnly": "npm test",
     "test": "ts-node test | tap-diff",
     "test:unit": "ts-node test/unit | tap-diff",
-    "test:e2e": "ts-node test/e2e",
-    "postinstall": "npm run build"
```

Comment:
@zippy @Connoropolous - Removing this is causing the CI's to fail. Is it a problem to have this in?

Reply:
@zo-el 

What I propose is that `test/e2e` is re-added, but "postinstall" isn't. My thinking is that, if we just get this approved, then we can build&release this, and we don't need "postinstall" , except for if tryorama goes back into development mode and you need to point all your stuff to a branch of it. I don't think "postinstall": "npm run build" should be in the main branch though, because it will force an extra "npm run build" step everytime that it gets "npm install"ed

Reply:
I thought I heard @Connoropolous say that the postinstall shouldn't stay in for release versions.  So I don't know for sure.  I deffer to you.